### ISSUE TITLE
Requires containers >= 0.5

### ISSUE THREAD

**Issue by phadej** [ndmitchell/hoogle#177]
```
Dependency containers ==0.4.2.1: using containers-0.4.2.1
...
/opt/ghc/7.4.2/bin/ghc --make -package-name hoogle-5.0 ....
src/Input/Cabal.hs:25:18:
    Could not find module `Data.Map.Strict'
```

**Comment by ndmitchell** [comment]
Thanks for the report, I'll fix that.

However, do note that Warp doesn't support GHC 7.4 so you aren't going to be able to build Hoogle anyway. GHC 7.6 is the minimum.

**Comment by phadej** [comment]
FWIW: `warp` as specifed by `hoogle-5.0` bounds *does* support GHC-7.4. There are a lot of buildable `warp-3.0.*` versions, for example. Please specify both lower and upper bounds.

(*Note* old hoogle versions bitrot because of unqualified imports:
```hs
import Development.Shake
import Development.Shake.FilePath
import CmdLine.All
```

```
src/Recipe/Command.hs:12:9:
    Ambiguous occurrence ‘CmdLine’
    It could refer to either ‘Development.Shake.CmdLine’,
                             imported from ‘Development.Shake’ at src/Recipe/Command.hs:7:1-24
                             (and originally defined in ‘Development.Shake.Command’)
                          or ‘CmdLine.All.CmdLine’,
                             imported from ‘CmdLine.All’ at src/Recipe/Command.hs:9:1-18
                             (and originally defined in ‘CmdLine.Type’
                                at src/CmdLine/Type.hs:(19,1)-(57,35))
```

---

If you do so, please specify your dependencies precisely (PVP says: minor range, i.e. `foo ==1.2.3.*`, as more new names can be added to any module).

**Comment by ndmitchell** [comment]
When I tried to install using GHC 7.4 I failed, with the output: https://travis-ci.org/ndmitchell/hoogle/jobs/143602815. Do I need some more constraints? Some other version ranges? I'm happy to enable it if I can test it.

I take a pragmatic approach to version bounds. If you want to use an older Hoogle I suggest you do in conjunction with Stackage, or by manually limiting the upper bounds to whatever the highest version was on the day I released the package. Otherwise, just use the latest Hoogle, which should always work with the latest versions.
